src/Grpc/HttpApi/test/Microsoft.AspNetCore.Grpc.HttpApi.IntegrationTests/Microsoft.AspNetCore.Grpc.HttpApi.IntegrationTests.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <IsPackable>false</IsPackable>
+    <Nullable>enable</Nullable>
+
+    <RestoreAdditionalProjectSources>$(RestoreAdditionalProjectSources);$(ArtifactsShippingPackagesDir)</RestoreAdditionalProjectSources>
+    <UseAspNetCoreSharedRuntime>true</UseAspNetCoreSharedRuntime>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\Shared\TestGrpcServiceActivator.cs" Link="Infrastructure\TestGrpcServiceActivator.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\Microsoft.AspNetCore.Grpc.HttpApi\Microsoft.AspNetCore.Grpc.HttpApi.csproj" />
```

Comment:
Please do not use `@(ProjectReference)` items for src/ projects. Instead,

1. run eng/scripts/GenerateProjectList.ps1 in this branch
2. add the updated eng/ProjectReferences.props file to a new commit
3. use (simpler) `@(Reference)` items instead e.g. `<Reference Include="Microsoft.AspNetCore.Grpc.HttpApi" />`

Reply:
I ran `GenerateProjectList.ps1` and `ProjectReferences.props` wasn't updated. Any idea why?